docs/upgrading_to_v36.0.md

```diff
@@ -25,6 +25,91 @@ The variable `logging_variant` has been removed from the Autopilot sub-modules.
 }
 ```
 
+### ACM Sub-Module Removal
+The ACM Sub-Module has been removed in v36.0.  Please use the [google_gke_hub_feature](https://registry.terraform.io/providers/hashicorp/google/latest/docs/resources/gke_hub_feature#example-usage---enable-fleet-default-member-config-configmanagement) and [google_gke_hub_feature_membership](https://registry.terraform.io/providers/hashicorp/google/latest/docs/resources/gke_hub_feature_membership#example-usage---config-management-with-git) resources.  For additional examples, see [terraform-docs-samples/gke/autopilot/config_sync](https://github.com/terraform-google-modules/terraform-docs-samples/tree/main/gke/autopilot/config_sync) and [terraform-docs-samples/gke/autopilot/policycontroller](https://github.com/terraform-google-modules/terraform-docs-samples/tree/main/gke/autopilot/policycontroller).
+
+
+```diff
+-module "acm" {
+-  source  = "terraform-google-modules/kubernetes-engine/google//modules/acm"
+-  version = "~> 35.0"
+-  project_id   = var.project_id
+-  location     = module.gke.location
+-  cluster_name = module.gke.name
+
+-  # Config Sync
+-  enable_config_sync = true
+-  sync_repo   = "git@github.com:GoogleCloudPlatform/anthos-config-management-samples.git"
+-  sync_branch = "1.0.0"
+-  policy_dir  = "foo-corp"
+
+-  # Policy Controller
+-  enable_policy_controller = true
+-  enable_fleet_feature = true
+-  install_template_library = true
+-  enable_referential_rules = true
+-  policy_bundles = ["https://github.com/GoogleCloudPlatform/acm-policy-controller-library.git/bundles/pss-baseline-v2022"]
+-}
+
+# Config Sync
++resource "google_gke_hub_feature" "config_feature" {
++  project  = var.project_id
++  location = "global"
++  name     = "servicemesh"
++}
```

Comment:
Should it be something like this:
https://github.com/terraform-google-modules/terraform-docs-samples/blob/main/gke/autopilot/config_sync/git/main.tf#L35-L38

Reply:
Thanks @janetkuo! Yes, I based this off a migration guide I previously wrote for ASM (prior create the sample you linked) and clearly missed updating this name field.